SPARCore/GADArticle.m

```diff
@@ -2,6 +2,48 @@
 
 @implementation GADArticle
 
++ (void) fetchArticlesWithPublication: (NSString *)publicationId
+                                        completionHandler:(void(^_Nonnull)(NSArray<GADRemoteModel *>
+                                                                           *_Nullable models,
+                                                            NSError *_Nullable error))completion {
+    //Should we create NSDictionary parameters within each method and just request the publicationId
+    //from the front end?
+
+}
+
+- (void) populateArticleWithId: (NSString *)articleId
```

Comment:
The idea behind this method would be to take a currently existing GADArticle object and populate the rest of its fields with the full article request, so it would be adding... what looks like the dateEdited and content fields. 
I'm consulting https://github.com/GrinnellAppDev/publications-admin/blob/master/src/app/state/models.ts and looking at the ShortArticleModel and the FullArticleModel, which is where I'm drawing that conclusion from.